source/transfer_list.rst

```diff
@@ -245,10 +245,11 @@ Anyone is free to use tags from that range for any custom TE layout without
 adding their definitions to this specification first. The use of this range is
 *strongly discouraged* for anything other than local experiments or code that
 will only ever be used in closed-source components owned by the entity
-controlling the entire final firmware image. In particular, a TE just
-containing platform-specific data or internal structures specific to a single
-firmware implementation is no reason not to allocate a standardized tag for it
-in this specification. Since standards often emerge organically, the goal is to
+controlling the entire final firmware image. In particular, a creator of a TE
+that just contains platform-specific data or internal structures specific to
+a single firmware implementation, should allocate a standardized tag for it
+in this specification -- using the non-standardized range is strongly
+discouraged. Since standards often emerge organically, the goal is to
```

Comment:
This statement repeats what is already said above so should be removed: "using the non-standardized range is strongly discouraged."

Reply:
Done